ansible_navigator/command_runner/command_runner.py

```diff
@@ -0,0 +1,102 @@
+""" command runner """
+import multiprocessing
+from queue import Queue
+import subprocess
+
+from typing import Any
+from typing import Callable
+from typing import List
+from typing import Union
+
+from types import SimpleNamespace
+
+PROCESSES = (multiprocessing.cpu_count() - 1) or 1
+
+
+class Command(SimpleNamespace):
```

Comment:
Can't we use the [CommandRunner](https://github.com/ansible/ansible-navigator/blob/main/ansible_navigator/runner/api.py#L258) and/or [CommandRunnerAsync](https://github.com/ansible/ansible-navigator/blob/main/ansible_navigator/runner/api.py#L229) from runner API's here?

I am planning to handle splitting the classes into different files as part of the issue https://github.com/ansible/ansible-navigator/issues/299

Reply:
we could....  I was thinking we would leave runner to do just do EE and ansible business, which is why I added this here, so ansible-navigator had some basic subprocess mode functionality independent of runner

Reply:
See #343